src/__tests__/resources/assetList.integration.specs.ts

```diff
@@ -0,0 +1,82 @@
+// Copyright 2019 Cognite AS
+
+import { CogniteClient } from '../..';
+// tslint:disable-next-line:no-commented-code
+// import { EventsAPI } from '../../resources/events/eventsApi';
+// import { TimeSeriesAPI } from '../../resources/timeSeries/timeSeriesApi';
+// import { CogniteEvent, GetTimeSeriesMetadataDTO } from '../../types/types';
+import { sleepPromise } from '../../utils';
+import { setupLoggedInClient } from '../testUtils';
+
+describe('assetList', () => {
+  let client: CogniteClient;
+  beforeAll(async () => {
+    client = setupLoggedInClient();
+  });
+  test('get timeseries', async () => {
+    // tslint:disable-next-line:no-commented-code
+    // const fetchedResources = getResources(client, client.timeseries);
+    // await client.timeseries.delete(fetchedResources.map(timeserie => ({ id: timeserie.id })));
+
+    const assets = [{ name: 'First asset' }, { name: 'Second asset' }];
+    const createdAssets = await client.assets.create(assets);
+    const timeseries = [
+      { name: 'Pressure sensor', assetId: createdAssets[0].id },
+      { name: 'Temprature sensor', assetId: createdAssets[1].id },
+    ];
+    const createdTimeseries = await client.timeseries.create(timeseries);
+    await sleepPromise(2000); // Backend issues
+    const timeSeriesFromAssetList = await createdAssets.timeSeries();
+    expect(timeSeriesFromAssetList.length).toBe(createdTimeseries.length);
+    await client.assets.delete(createdAssets.map(asset => ({ id: asset.id })));
+    await client.timeseries.delete(
+      createdTimeseries.map(timeserie => ({ id: timeserie.id }))
+    );
+  });
+
+  test('get events', async () => {
+    // tslint:disable-next-line:no-commented-code
+    // const fetchedResources = getResources(client, client.events);
+    // await client.events.delete(fetchedResources.map(event => ({ id: event.id })));
+    const assets = [{ name: 'First asset' }, { name: 'Second asset' }];
+    const createdAssets = await client.assets.create(assets);
+    const events = [
+      { description: 'Test event 1', assetIds: [createdAssets[0].id] },
+      { description: 'Test event 2', assetIds: [createdAssets[1].id] },
+    ];
+    const createdEvents = await client.events.create(events);
+    await sleepPromise(2000); // Backend issues
+    const eventFromAssetList = await createdAssets.events();
+    expect(eventFromAssetList.length).toBe(createdEvents.length);
+    await client.assets.delete(createdAssets.map(asset => ({ id: asset.id })));
+    await client.events.delete(createdEvents.map(event => ({ id: event.id })));
+  });
+});
+
+// Help method for testing that is not really pretty, cleaner to just keep the code above?
+
+// tslint:disable-next-line:no-commented-code
```

Comment:
Remove comments

Reply:
@f1cognite All these comments are because of the helping method. Should I go for the helping method or for the way it is now (and remove the comments)?